### ISSUE TITLE
accuracy question

### ISSUE THREAD

**Issue by TimoLeung** [mayenkk/SkinSage#2]
I notice that your trained model achieves an accuracy score of 91.7% on the test set, it doesn't fit the "training and validation accuracy" figure on the code. Otherwise, on my training with 20 epoches, it achieves 93.76% accuracy on the training set and 47.74% on the test set. I've spend a long time to find out why, but it doesn't solved yet, can you help me check with that? I appreciate that !!
[skinsage.txt](https://github.com/user-attachments/files/16540521/skinsage.txt)
